docs/help/How-To-Report-Issues-Helpfully.html

```diff
@@ -21,12 +21,13 @@ <h3>Reporting The Issue</h3>
 <p>Once you know which type of issue yours is, it's time to report it. It's best to report issues on LOOT's issue trackers, where they can't get lost, overlooked or disorganised, but doing so requires a free GitHub account. The links below can be used to report your issue in the appropriate tracker.</p>
 
 <ul>
-  <li><a href="https://github.com/loot/oblivion/issues/new">Oblivion metadata</a></li>
+  <li><a href="https://github.com/loot/morrowind/issues/new">Morrowind metadata</a></li>
```

Comment:
Could you also add https://github.com/loot/enderal/issues/new for Enderal?

Reply:
@Ortham I didn't add any mention of Enderal to https://loot.github.io/ since it's not supported in v0.16.0, & the website can't distinguish between the latest stable release & the latest source code like the other docs can. I thought it'd cause some confusion if people see that LOOT supports Enderal, but it doesn't in the latest release. Do you want me to add it anyway?

Reply:
Ah, fair enough. No, leave it out for now then.